Cargo.toml

```diff
@@ -14,6 +14,7 @@ tracing-subscriber = { version = "0.3", default-features = false, features = ["r
 nu-ansi-term = "0.46.0"
 atty = "0.2"
 tracing-log = { version = "0.1", optional = true }
+time = { version = "0.3.20", features = ["formatting", "local-offset"] }
```

Comment:
Seems like a heavy dependency, to be honest. Maybe it's justified to move it under a feature flag?

Reply:
Now it's optional.